src/AddTodoForm.js

```diff
@@ -1,18 +1,38 @@
 import React from 'react';
 
+// this file renders the form where each todo item is entered to create the todolist
+
 function AddTodoForm(props) {
```

Comment:
I see you're deconstructing your pops on line 7. Nice job!

You can actually deconstruct the props in the AddTodoForm function's signature. Like this:
`function AddTodoForm( {onAddTodo} ){`

What do you think of this?